arviz/stats/diagnostics.py

```diff
@@ -65,7 +65,16 @@ def bfmi(data):
     return _bfmi(dataset.energy)
 
 
-def ess(data, *, var_names=None, method="bulk", relative=False, prob=None):
+def ess(
+    data,
+    *,
+    var_names=None,
+    method="bulk",
+    relative=False,
+    prob=None,
+    dask="forbidden",
+    output_dtypes=None
```

Comment:
it would be better to use `**kwargs`, this way we don't have to keep the docstrings and defaults up to date with xarray (and we can directly link to their docs: https://arviz-devs.github.io/arviz/developer_guide.html#documentation) and if extra kwargs were needed (or if new arguments were added by xarray) everything would still work

Reply:
Agreed

Reply:
I thought of that too but then got confused by the `kwargs` argument used by `xr.apply_ufunc`, now I know the difference between `kwargs` and `**kwargs` :sweat_smile: